docs/en/stack/ml/df-analytics/supervised-lifecycle.adoc

```diff
@@ -0,0 +1,11 @@
+[ditaa, target="ml-dfa-lifecycle-diagram"]
+....
+                            
++-----------+   +-----------+   +-----------+   +-----------+
+|           |-->+           +-->+           +-->|           |
```

Comment:
Very minor issue, but the first arrow looks different than the others, so I suggest making it the same:
```suggestion
|           +-->+           +-->+           +-->|           |
```